ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -115,6 +115,30 @@ conformance_test(
     extra_runner_args = ["7000"],
     lf_versions = [
         "default",
+        #        "preview", # FIXME: enable once merged with `conformance-test-dev-domain-preview-dev`
+        #        "dev", # FIXME: enable once merged with `conformance-test-dev-domain-preview-dev`
+    ],
+    ports = conformance_test_ports,
+    preview_mod_flag = "",
+    runner = "@//bazel_tools/client_server/runner_with_health_check",
+    server = ":canton-test-runner-with-dependencies",
+    server_args = [
+        "--config=$$(canonicalize_rlocation $(rootpath :canton-dev.conf))",
+        "--bootstrap=$$(canonicalize_rlocation $(rootpath :bootstrap-dev.canton))",
+    ],
+    test_tool_args = conformance_test_tool_args,
+) if not is_windows else None
+
+conformance_test(
+    name = "conformance-test-dev-domain-preview-dev",
```

Comment:
Needed in order to allow the exclusion of the test whose error code has changed. This change will be reverted once Canton catches up with the code change from SDK that changes the error code